tree/ntuple/v7/inc/ROOT/RNTupleReader.hxx

```diff
@@ -305,6 +305,39 @@ public:
       return GetView<T>(RetrieveFieldId(fieldName), rawPtr);
    }
 
+   /// Provides access to an individual (sub)field from its on-disk ID.
```

Comment:
There are many spurious extra changes that somehow ended up in this PR, make sure to remove them

Reply:
it's actually a full file...